version.sbt

```diff
@@ -1 +1 @@
-version in ThisBuild := "2015.06.19-1-SNAPSHOT"
\ No newline at end of file
+version in ThisBuild := "0.0.1"
```

Comment:
> Yes, because the date versioning was provided by the plugin I removed

Ah right. SGTM, though an initial version of 0.0.1 "feels" slightly wrong. It is all very arbitrary, but when I see 0.0.1 I think this is a pre-alpha-release. I don't think we've talked about an AI2 policy for semantic versioning (i.e. when we consider something worthy of 1.0 status). Another idea is to change our group id + artifact id at this time to `"org.allenai" % "plugins" % VERSION` to enforce a clean separation from our date-based versioned artifacts.

Thoughts?

BTW, feel free to merge with 0.0.1 version. I don't have strong feelings but this seems like a good opportunity to discuss AI2 semantic versioning policy.